src/one_dragon_qt/widgets/color_tip.py

```
@@ -0,0 +1,190 @@
+
+
+class ColorInfoWidget(QWidget):
+    """
+    单个颜色信息显示控件
+    """
+
+    def __init__(self, color_info: Dict[str, Any], parent: Optional[QWidget] = None):
+        """
+        初始化颜色信息控件
+
+        :param color_info: 颜色信息字典，包含pos, rgb, hsv等信息
```

Ruff (0.14.2)

Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)